client/src/components/infoDrawer/infoDrawer.js

```diff
@@ -49,14 +80,46 @@ class InfoDrawer extends PureComponent {
         onClose={this.handleClose}
         {...{ isOpen, position }}
       >
-        <InfoFormat
-          {...{
-            datasetTitle,
-            aboutURL,
-            singleValueCategories,
-            dataPortalProps,
-          }}
-        />
+        <Async
+          watchFn={InfoDrawer.watchAsync}
+          promiseFn={this.fetchAsyncProps}
+          watchProps={{ schema }}
+        >
+          <Async.Pending>
+            <InfoFormat
+              {...{
+                datasetTitle,
+                aboutURL,
+                singleValueCategories,
+                dataPortalProps,
+              }}
+            />
```

Comment:
We need to reintroduce the loading state for the InfoFormat component (the skeleton + placeholder generation)